packages/client/hmi-client/src/components/workflow/scenario-templates/scenario-template-utils.ts

```diff
@@ -0,0 +1,34 @@
+import { DistributionType } from '@/services/distribution';
+import { ParameterSemantic } from '@/types/Types';
+import { calculateUncertaintyRange } from '@/utils/math';
+
+export const displayParameter = (parameters: ParameterSemantic[], parameterName: string) => {
+	let value = '';
+	const parameter = parameters.find((p) => p.referenceId === parameterName);
+	switch (parameter?.distribution.type) {
+		case DistributionType.Constant:
+			value = `${parameter.distribution.parameters.value}`;
+			break;
+		case DistributionType.Uniform:
+			value = `${parameter.distribution.parameters.minimum} - ${parameter.distribution.parameters.maximum}`;
+			break;
+		default:
+			return '';
+	}
+	return `${parameterName}  [${value}]`;
+};
+
+export const formatUniformDistribution = (parameter: ParameterSemantic) => {
+	if (parameter.distribution.type !== DistributionType.Uniform) {
+		let minimum = 0;
+		let maximum = 1;
+		if (parameter.distribution.type === DistributionType.Constant) {
+			// +10% and -10% of the constant value
+			const { min, max } = calculateUncertaintyRange(parameter.distribution.parameters.value, 10);
+			minimum = min;
+			maximum = max;
```

Comment:
Note any percentage of 0 is still 0, you may want to default to [0, 1] if that is the case.

Reply:
Now I've added a check to see if min and max are 0, it will set max to 1